python/desc/sims_ci_pipe/ellipticity_distributions.py

```diff
@@ -34,15 +35,19 @@ def get_e(ixx, iyy, ixy):
     return (a**2 - b**2)/(a**2 + b**2)
 
 
-def get_point_sources(src, flux_type='base_PsfFlux'):
+def get_point_sources(src, flux_type='base_PsfFlux', flags=()):
```

Comment:
I think it also may be useful to have the ability to add a SNR cut.

Reply:
Done.